packages/edit-site/src/components/global-styles/shadows-panel.js

```diff
@@ -136,11 +195,12 @@ function ShadowItem( { shadow, category } ) {
 		<NavigationButtonAsItem
 			path={ `/shadows/edit/${ category }/${ shadow.slug }` }
 		>
-			<HStack>
-				<FlexItem>{ shadow.name }</FlexItem>
-				<FlexItem display="flex">
-					<Icon icon={ isRTL() ? chevronLeft : chevronRight } />
-				</FlexItem>
+			<HStack justify="space-between">
+				<HStack justify="flex-start">
+					<IconWithCurrentColor icon={ shadowIcon } />
```

Comment:
The `shadowIcon` was removed in https://github.com/WordPress/gutenberg/pull/67720 as per [this comment](https://github.com/WordPress/gutenberg/issues/67696#issuecomment-2525550554), so I don't think we should introduce it again here.

If we remove the `shadowIcon`, then I think we can also simplify the markup in line with [the changes](https://github.com/WordPress/gutenberg/pull/67720/files#diff-d76fe0b9848e786a8cc26155bd54635fb31ed7bb5ba193af1e702fe7c1cbeaaeR139) in the above PR (removing the additional `HStack`).

Reply:
This is now fixed.